web/src/main/java/hu/psprog/leaflet/lms/web/controller/DocumentsController.java

```diff
@@ -6,10 +6,7 @@
 import hu.psprog.leaflet.lms.service.facade.DocumentFacade;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Controller;
-import org.springframework.web.bind.annotation.ModelAttribute;
-import org.springframework.web.bind.annotation.PathVariable;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.*;
```

Comment:
Please don't use asterisk for imports.

Reply:
It will be fixed after IDE settings change.